src/modules/components/proposal/proposalDataListItem/approvalThresholdResult/approvalThresholdResult.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { ApprovalThresholdResult, type IApprovalThresholdResultProps } from './approvalThresholdResult';
+
+describe('<ApprovalThresholdResult /> component', () => {
+    const createTestComponent = (props?: Partial<IApprovalThresholdResultProps>) => {
+        const completeProps: IApprovalThresholdResultProps = {
+            approvalAmount: 1,
+            approvalThreshold: 2,
+            ...props,
+        };
+
+        return <ApprovalThresholdResult {...completeProps} />;
+    };
+
+    it('renders the given approval threshold, approval amount and corresponding progressbar', () => {
```

Comment:
It would be nice to test that the amounts are properly formatted and the progress value is correct. The progress value could be checked with `screen.getByRole('progressbar').getAttribute('data-value'))`

Reply:
The progressBar assertion makes sense but not the formatted one. It's in essence just checking if the formatter was used as opposed to whether it's properly formatted.

Reply:
Well yes, using the formatter to format the numbers is part of the component's logic, we could add some tests to make sure that the formatter is being used with the correct format. In theory you could assert this through a spy and then `.toHaveBeenCalledWith`, but it's just simpler and more straight forward to assert that a text is rendered on `screen`